crates/holochain/src/core/workflow/publish_dht_ops_workflow.rs

```diff
@@ -388,41 +299,22 @@ mod tests {
             // Create test env
             let test_env = test_cell_env();
             let env = test_env.env();
-            let env_ref = env.guard();
 
             // Setup
             let (_network, cell_network, recv_task, _) =
                 setup(env.clone(), num_agents, num_hash, true).await;
 
             // Update the authored to have > R counts
-            {
-                let reader = env_ref.reader().unwrap();
-                let mut workspace = PublishDhtOpsWorkspace::new(env.clone().into()).unwrap();
-
-                // Update authored to R
-                let values = workspace
-                    .authored_dht_ops
-                    .iter(&reader)
-                    .unwrap()
-                    .map(|(k, mut v)| {
-                        v.receipt_count = DEFAULT_RECEIPT_BUNDLE_SIZE;
-                        Ok((DhtOpHash::from_raw_39_panicky(k.to_vec()), v))
-                    })
-                    .collect::<Vec<_>>()
-                    .unwrap();
-
-                for (hash, v) in values.into_iter() {
-                    workspace.authored_dht_ops.put(hash, v).unwrap();
-                }
-
-                // Manually commit because this workspace doesn't commit to all dbs
-                env_ref
-                    .with_commit::<DatabaseError, _, _>(|writer| {
-                        workspace.authored_dht_ops.flush_to_txn(writer)?;
-                        Ok(())
-                    })
+            env.conn()
+                .unwrap()
+                .with_commit_test(|txn| {
+                    txn.execute(
+                        "UPDATE DhtOp SET receipt_count = ?",
```

Comment:
!! uhh... there's no WHERE here, isn't this going to update all of them?

Reply:
Ah, n/m I see this is in a test : )

Reply:
Yep this is a test and it's intentionally setting all receipts to the full count to check they don't get published once that count is reached.